metrics/xtreme_s/xtreme_s.py

```diff
@@ -0,0 +1,185 @@
+    try:
+        import sacrebleu as scb
+    except ImportError:
+        raise ValueError(
+            "sacrebleu has to be installed in order to apply the bleu metric for covost2."
+            "You can install it via `pip install sacrebleu`."
+        )
+
+    if version.parse(scb.__version__) < version.parse("1.4.12"):
+        raise ImportWarning(
+            "To use `sacrebleu`, the module `sacrebleu>=1.4.12` is required, and the current version of `sacrebleu` doesn't match this condition.\n"
+            'You can install it with `pip install "sacrebleu>=1.4.12"`.'
+        )
+
+    references_per_prediction = len(labels[0])
+    if any(len(refs) != references_per_prediction for refs in labels):
+        raise ValueError("Sacrebleu requires the same number of references for each prediction")
+    transformed_references = [[refs[i] for refs in labels] for i in range(references_per_prediction)]
+    output = scb.corpus_bleu(
+        preds,
+        transformed_references,
+        smooth_method=smooth_method,
+        smooth_value=smooth_value,
+        force=force,
+        lowercase=lowercase,
+        use_effective_order=use_effective_order,
+        **(dict(tokenize=tokenize) if tokenize else {}),
+    )
+    return {"bleu": output.score}
+
+
+def wer(preds, labels, concatenate_texts, config_name):
+    try:
+        from jiwer import compute_measures
+    except ImportError:
+        raise ValueError(
+            f"jiwer has to be installed in order to apply the wer metric for {config_name}."
+            "You can install it via `pip install jiwer`."
+        )
+    if concatenate_texts:
+        return {"wer": compute_measures(labels, preds)["wer"]}
+    else:
+        incorrect = 0
+        total = 0
+        for prediction, reference in zip(preds, labels):
+            measures = compute_measures(reference, prediction)
+            incorrect += measures["substitutions"] + measures["deletions"] + measures["insertions"]
+            total += measures["substitutions"] + measures["deletions"] + measures["hits"]
+        return {"wer": incorrect / total}
+
+
+@datasets.utils.file_utils.add_start_docstrings(_DESCRIPTION, _KWARGS_DESCRIPTION)
+class XtremeS(datasets.Metric):
+    def _info(self):
+        if self.config_name not in _CONFIG_NAMES:
+            raise KeyError(f"You should supply a configuration name selected in {_CONFIG_NAMES}")
+
+        pred_type = "int64" if self.config_name in ["fleurs", "minds14"] else "string"
+
+        return datasets.MetricInfo(
+            description=_DESCRIPTION,
+            citation=_CITATION,
+            inputs_description=_KWARGS_DESCRIPTION,
+            features=datasets.Features(
+                {"predictions": datasets.Value(pred_type), "references": datasets.Value(pred_type)}
+            ),
+            codebase_urls=[],
+            reference_urls=[],
+            format="numpy",
+        )
+
+    def _compute(self, predictions, references, bleu_kwargs, wer_kwargs):
```

Comment:
```suggestion
    def _compute(self, predictions, references, bleu_kwargs={}, wer_kwargs={}):
```
Allow falling back to default kwargs values

Reply:
Think mutable objects in function signatures is always a bit tricky. Could we maybe point it to `None` and then if None set to a dict?

Reply:
Sure, that works too :)